src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit_base.html.twig

```diff
@@ -538,10 +538,16 @@
 
 {% block date_range_widget %}
   {% apply spaceless %}
-    {{ form_widget(form.from) }}
-    {{ form_errors(form.from) }}
-    {{ form_widget(form.to) }}
-    {{ form_errors(form.to) }}
+    <div class="date-range">
+      {{ form_widget(form.from, { 'attr': {'class': 'from'} }) }}
+      {{ form_errors(form.from) }}
+      {{ form_widget(form.to) }}
+      {{ form_errors(form.to) }}
+      {% if form.unlimited is defined %}
```

Comment:
This seems ok but I'm afraid this won't match the expected mockup for product page, or does it?

Since the unlimited checkbox is expected to be displayed under the `form.to` field I would expect some additional divs to handle the columns or flex rendering

But it's specific to the specific price, so maybe you should keep this one generic as is but override this template in the specific price form theme to handle the custom display?